workshops/models.py

```diff
@@ -332,7 +332,10 @@ def clean(self):
         """This will be called by the ModelForm.is_valid(). No saving to the
         database."""
         # lowercase the email
-        self.email = self.email.lower()
```

Comment:
@narayanaditya95  Then it should be : `self.email = self.email.lower() if self.email else None`